From 5bf9dfe906b50f2455210a29a00ec5494d8e26b7 Mon Sep 17 00:00:00 2001
From: Jack Ostroff <ostroffjh@users.sourceforge.net>
Date: Thu, 30 Aug 2018 19:02:11 -0400
Subject: Correcting symlinks between col- and row-resize and split_v and _h
 cursors

Summary: BUG: 384864

Reviewers: #plasma, apol, cfeck

Reviewed By: cfeck

Subscribers: davidedmundson, rdieter, tfoster, apol, asturmlechner, cfeck, plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D8267
---
 cursors/KDE_Classic/cursors/col-resize   | 2 +-
 cursors/KDE_Classic/cursors/row-resize   | 2 +-
 cursors/Oxygen_Black/cursors/col-resize  | 2 +-
 cursors/Oxygen_Black/cursors/row-resize  | 2 +-
 cursors/Oxygen_Blue/cursors/col-resize   | 2 +-
 cursors/Oxygen_Blue/cursors/row-resize   | 2 +-
 cursors/Oxygen_White/cursors/col-resize  | 2 +-
 cursors/Oxygen_White/cursors/row-resize  | 2 +-
 cursors/Oxygen_Yellow/cursors/col-resize | 2 +-
 cursors/Oxygen_Yellow/cursors/row-resize | 2 +-
 cursors/Oxygen_Zion/cursors/col-resize   | 2 +-
 cursors/Oxygen_Zion/cursors/row-resize   | 2 +-
 cursors/src/symlinks/col-resize          | 2 +-
 cursors/src/symlinks/row-resize          | 2 +-
 14 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/cursors/KDE_Classic/cursors/col-resize b/cursors/KDE_Classic/cursors/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/KDE_Classic/cursors/col-resize
+++ b/cursors/KDE_Classic/cursors/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/KDE_Classic/cursors/row-resize b/cursors/KDE_Classic/cursors/row-resize
index e3aa748..179b52b 120000
--- a/cursors/KDE_Classic/cursors/row-resize
+++ b/cursors/KDE_Classic/cursors/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
diff --git a/cursors/Oxygen_Black/cursors/col-resize b/cursors/Oxygen_Black/cursors/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/Oxygen_Black/cursors/col-resize
+++ b/cursors/Oxygen_Black/cursors/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/Oxygen_Black/cursors/row-resize b/cursors/Oxygen_Black/cursors/row-resize
index e3aa748..179b52b 120000
--- a/cursors/Oxygen_Black/cursors/row-resize
+++ b/cursors/Oxygen_Black/cursors/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
diff --git a/cursors/Oxygen_Blue/cursors/col-resize b/cursors/Oxygen_Blue/cursors/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/Oxygen_Blue/cursors/col-resize
+++ b/cursors/Oxygen_Blue/cursors/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/Oxygen_Blue/cursors/row-resize b/cursors/Oxygen_Blue/cursors/row-resize
index e3aa748..179b52b 120000
--- a/cursors/Oxygen_Blue/cursors/row-resize
+++ b/cursors/Oxygen_Blue/cursors/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
diff --git a/cursors/Oxygen_White/cursors/col-resize b/cursors/Oxygen_White/cursors/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/Oxygen_White/cursors/col-resize
+++ b/cursors/Oxygen_White/cursors/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/Oxygen_White/cursors/row-resize b/cursors/Oxygen_White/cursors/row-resize
index e3aa748..179b52b 120000
--- a/cursors/Oxygen_White/cursors/row-resize
+++ b/cursors/Oxygen_White/cursors/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
diff --git a/cursors/Oxygen_Yellow/cursors/col-resize b/cursors/Oxygen_Yellow/cursors/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/Oxygen_Yellow/cursors/col-resize
+++ b/cursors/Oxygen_Yellow/cursors/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/Oxygen_Yellow/cursors/row-resize b/cursors/Oxygen_Yellow/cursors/row-resize
index e3aa748..179b52b 120000
--- a/cursors/Oxygen_Yellow/cursors/row-resize
+++ b/cursors/Oxygen_Yellow/cursors/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
diff --git a/cursors/Oxygen_Zion/cursors/col-resize b/cursors/Oxygen_Zion/cursors/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/Oxygen_Zion/cursors/col-resize
+++ b/cursors/Oxygen_Zion/cursors/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/Oxygen_Zion/cursors/row-resize b/cursors/Oxygen_Zion/cursors/row-resize
index e3aa748..179b52b 120000
--- a/cursors/Oxygen_Zion/cursors/row-resize
+++ b/cursors/Oxygen_Zion/cursors/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
diff --git a/cursors/src/symlinks/col-resize b/cursors/src/symlinks/col-resize
index 179b52b..e3aa748 120000
--- a/cursors/src/symlinks/col-resize
+++ b/cursors/src/symlinks/col-resize
@@ -1 +1 @@
-split_v
\ No newline at end of file
+split_h
\ No newline at end of file
diff --git a/cursors/src/symlinks/row-resize b/cursors/src/symlinks/row-resize
index e3aa748..179b52b 120000
--- a/cursors/src/symlinks/row-resize
+++ b/cursors/src/symlinks/row-resize
@@ -1 +1 @@
-split_h
\ No newline at end of file
+split_v
\ No newline at end of file
-- 
cgit v0.11.2
